docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-tags/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Snowplow v3 Tags"
+sidebar_position: 1000
+---
+
+Snowplow provides two custom GTM Tag templates to use:
```

Comment:
Should we avoid the number "two" so this doesn't have to be maintained if we add more?